cmd/lotus-shed/miner-fees.go

```diff
@@ -148,9 +165,172 @@ var minerFeesCmd = &cli.Command{
 				correct = "✗"
 				discrepancies = true
 			}
-			_, _ = fmt.Fprintf(cctx.App.Writer, "\t%s Deadline daily fee: %s (sector fee sum: %s, expiration fee deduction sum: %s)\n", correct, dl.DailyFee, totalSectorsFee, totalFeeDeduction)
+			_, _ = fmt.Fprintf(cctx.App.Writer, "\t%s Deadline daily fee: %s, power: %s (sector fee sum: %s, expiration fee deduction sum: %s)\n", correct, dl.DailyFee, dl.LivePower.QA, totalSectorsFee, totalFeeDeduction)
+
+			if !dl.DailyFee.IsZero() {
+
+				/** Look for evidence of fee payment **/
+
+				totalMinerFee = big.Add(totalMinerFee, dl.DailyFee)
+
+				// Epoch at which the deadline ends and we expect to find cron execution
+				dlEndEpoch := dlinfo.PeriodStart + abi.ChainEpoch(dlIdx+1)*dlinfo.WPoStChallengeWindow - 1
+				if dlEndEpoch > dlinfo.CurrentEpoch {
+					dlEndEpoch -= dlinfo.WPoStProvingPeriod
+				}
+				// If there is a null epoch involved we don't want to go backward, so we go AfterHeight
+				dlEndTs, err := api.ChainGetTipSetAfterHeight(ctx, dlEndEpoch, types.EmptyTSK)
+				if err != nil {
+					return xerrors.Errorf("getting tipset at deadline end epoch: %w", err)
+				}
+				// StateCompute at the height we arived at, which may not be dlEndEpoch if nulls were involved
+				compute, err := api.StateCompute(ctx, dlEndTs.Height(), nil, dlEndTs.Key())
+				if err != nil {
+					return xerrors.Errorf("computing tipset at deadline end epoch: %w", err)
+				}
+
+				_, _ = fmt.Fprintf(cctx.App.Writer, "\tInspecting last deadline cron @ %d:\n", dlEndTs.Height())
```

Comment:
handle error maybe?  What do we have to lose

Reply:
There's 14 of them in this file, so we lose 14 x 2 = 28 more lines in this file of boilerplate that won't be triggered, I could write these as `fmt.Printf` for the same effect and then get to pretend there's never an error

Alternatively we wait for this so we can just put a `?` on the end of the line: https://github.com/golang/go/issues/71460

Also:

```
$ git grep =\ fmt.Fprintf.*ctx.App | wc
     70     758    7863
$ git grep =\ fmt.Fprintf.*ctx.App | grep -v '\s*_, _' | wc
      0       0       0
```